cookbooks/aws-parallelcluster-platform/kitchen.platform-install.yml

```diff
@@ -209,3 +209,26 @@ suites:
       cluster:
         nvidia:
           enabled: yes
+  - name: nvidia_gdrcopy
+    # doesn't work on Docker
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-tests::test_resource]
+    verifier:
+      controls:
+        - /tag:install_.*gdrcopy/
+    driver:
+      # nvidia_driver can be executed only on a graphic EC2 instance example: g5.xlarge(x86_86) or g5g.xlarge(aarm64)
+      instance_type: g4dn.2xlarge
+    attributes:
+      resource: gdrcopy
+      dependencies:
+        - resource:package_repos
+        - resource:install_packages:install_kernel_source
+        - 'resource:package { "package_name": "dkms" }'
```

Comment:
q: dkms package is already included on gdrcopy deps, why do we need to explicitly add it here?
Are we missing it as deps of nvidia?

Reply:
This is just a copy/paste from other nvidia resources. I guess we can remove this line.